src/main/java/easytests/api/v1/models/User.java

```diff
@@ -0,0 +1,29 @@
+package easytests.api.v1.models;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import java.util.List;
+import lombok.Data;
+
+
+/**
+ * @author SvetlanaTselikova
+ */
+@Data
+public class User {
+    private Integer id;
+
+    private String firstName;
+
+    private String lastName;
+
+    private String surname;
+
+    private String email;
+
+    private Boolean isAdmin;
+
+    private Integer state;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private List<Identity> subjects;
```

Comment:
There is no need for subjects here, remove pls. I've just make an example to test how it works.